blog/tests.py

```diff
@@ -118,3 +132,172 @@ def test_delete_blog(self):
         force_authenticate(request, user=user)
         response = view(request, slug=blog.slug)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+    def test_update_blog(self):
+        """
+        Tests to update a Blog Post Successfully.
+        """
+        response = self.create_blog()
+        user = User.objects.get(username='test')
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        blog = Blog.objects.get()
+
+        body = {
+            'title': 'Test101',
+            'description': 'This is a Test Blog. Amirite! :D'
+        }
+
+        factory = APIRequestFactory()
+        url = '/api/blog/'
+
+        request = factory.put(url, body)
+        view = BlogAPI.as_view({
+            'put': 'update'
+        })
+
+        force_authenticate(request, user=user)
+        response = view(request, slug=blog.slug)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Blog.objects.count(), 1)
+        self.assertEqual(Blog.objects.get().title, 'Test101')
```

Comment:
use minimal hardcoded content. We can get the expected value from `blog_update_content` declared above.

Also try to put the expected value always first then compare it with the dynamic response value.